src/euphorie/content/choice.py

```diff
@@ -0,0 +1,84 @@
+from .. import MessageFactory as _
+from .behaviour.richdescription import IRichDescription
+from .fti import ConditionalDexterityFTI
+from .fti import IConstructionFilter
+from Acquisition import aq_chain
+from Acquisition import aq_inner
+from euphorie.content.survey import ISurvey
+from plone.app.dexterity.behaviors.metadata import IBasic
+from plone.app.vocabularies.catalog import StaticCatalogVocabulary
+from plone.autoform import directives
+from plone.dexterity.content import Container
+from plone.supermodel import model
+from z3c.form.browser.select import SelectFieldWidget
+from z3c.relationfield.schema import RelationChoice
+from z3c.relationfield.schema import RelationList
+from zope import schema
+from zope.component import adapter
+from zope.interface import implementer
+from zope.interface import Interface
+
+
+class IChoice(model.Schema, IRichDescription, IBasic):
+    """ """
+
+    allow_multiple_options = schema.Bool(
+        title=_("label_allow_multiple_options", default="Allow multiple options"),
+        description=_(
+            "help_allow_multiple_options",
+            default="If active, checkboxes are shown to allow selecting more than one "
+            "option. Otherwise, radio buttons are used to force a single selection.",
+        ),
+        default=True,
+    )
+    condition = RelationList(
+        title="Condition",
+        description="Only show this choice if the user selects certain options for "
+        "another choice. You can pick multiple options here. If the user selects one "
+        "or more of them, then this choice will be shown. Leave blank to always show "
+        "this choice.",
+        value_type=RelationChoice(
+            vocabulary=StaticCatalogVocabulary({"portal_type": ["euphorie.option"]}),
+        ),
+        required=False,
+        default=[],
+    )
+    directives.widget("condition", SelectFieldWidget)
```

Comment:
Is this querying the whole site?
Will the select widget end up having potentially thousands of options to select from?

Reply:
Yes. That's something on my list. But at least it's possible to select something like this.

Reply:
Done.